### PR TITLE
Add instruction for changing Composer version

### PR DESCRIPTION
**Pull request by shiftedreality**
## Purpose of this pull request

This pull request (PR) adds missing instruction on changing the Composer version for the latest Magento versions

### CODE REVIEW

File: src/cloud/docker/docker-config.md [hunk 1]
```diff
@@ -53,6 +53,14 @@ Alternatively, you can run the following command to add it to the file:
 echo "127.0.0.1 magento2.test" | sudo tee -a /etc/hosts
 ```
 
+## Change Composer version
+
+You can change Composer version by adding next line in `.docker/config.env` configuration file:
```
Comment (BarnyShergold) [inline src/cloud/docker/docker-config.md:58]:
```suggestion
You can change Composer version by adding the following line to the `.docker/config.env` configuration file:
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Small grammar suggestion

**Review by bdenham — APPROVED** [review]

**Comment by bdenham** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@bdenham the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8893`.

**Comment by bdenham** [comment]
@shiftedreality, this PR was imported to our devdocs team repo for merging and publishing: https://github.com/magento-commerce/devdocs/pull/2244.

**Pull request closed without merging**